Luke Smith Portfolio.css

```diff
@@ -0,0 +1,111 @@
+/** {
+	box-sizing: border-box;
+}
+*/
+body {
+  margin: auto;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.topnav {
+  overflow: auto;
+  background-color: #ffffff;
+}
+
+.topnav a {
+  float: left;
+  color: #000000;
+  text-align: center;
+  padding: 14px 16px;
+  text-decoration: none;
+  font-size: 17px;
+  width: auto;
+}
+
+.topnav a:hover {
+  background-color: #dddddd;
+  color: #000000;
+}
+
+body {
+  background: #555555;
+}
+
+
+/*About Me Section*/
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  color: black;
+  margin: 0;
+  max-width: 500px;
+  margin: auto;
+  padding: 10px;
+}
+
+.column {
+  float: left;
+  width: 33.3%;
+  margin-bottom: 16px;
+  padding: 0 8px;
+}
+
+.card {
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2);
+  margin: auto;
+}
+
+.about-section {
+  padding: 50px;
+  text-align: center;
+ 
+}
```

Comment:
about-section is another class I can't find in your html. Is this supposed to be applying to the bio on your "About Me" page? It looks as though the styling for that is being applied by another rule at the moment, so this can be safely deleted.